atef/widgets/config.py

```diff
@@ -1437,11 +1437,18 @@ def _open_device_chooser(self, to_select: Optional[List[str]] = None):
         to_select : list of str, optional
             If provided, the device chooser will filter for these items.
         """
-        self._search_widget = HappiSearchWidget(client=util.get_happi_client())
-        self._search_widget.happi_items_chosen.connect(self.add_items)
+        self._search_widget = HappiDeviceComponentWidget(
+            client=util.get_happi_client(),
+            show_device_components=False,
+        )
+        self._search_widget.item_search_widget.happi_items_chosen.connect(
+            self.add_items
+        )
         self._search_widget.show()
         self._search_widget.activateWindow()
-        self._search_widget.edit_filter.setText(util.regex_for_devices(to_select))
+        self._search_widget.item_search_widget.edit_filter.setText(
+            util.regex_for_devices(to_select)
```

Comment:
Note to self: functionally what this does for the device selector is allow us to double-click a device to view its metadata